WKData/Sources/WKData/Extensions/Int+HttpStatusCodes.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension Int {
+    var isHttpSuccess: Bool {
```

Comment:
I like the intention, and appreciate it only internal to `WKData`, but I wonder if this really needs to extend _all_ `Int`s over just being in some generalized HTTP status code utility.

Reply:
Good idea - I moved this into a `HTTPURLResponse` extension.